src/site/includes/metatags.drupal.liquid

```diff
@@ -56,4 +60,5 @@
 <meta content="{{ description }}" name="description">
   {% endif %}
 <meta content="{{ hostUrl }}/img/design/logo/va-og-image.png" property="og:image">
+<title>{{ metaTitle }}</title>
```

Comment:
I'm probably missing it, but should there be an `elsif` before this last title tag? Seeing a title tag also on line 12.

Reply:
No, the first one's in the "if tagsSize > 0" block. This one's in the else for that if.